### DISCUSSION TITLE
Does hydromt averages out the hourly values of temperature and radiation parameters?

### DISCUSSION THREAD

**Discussion by abdulrehman565** [Deltares/hydromt#384] [Q&A] [answered]
does hydromt takes the average of entire day for temperature and other radiation parameters in order to calculate pet? And what if we give daily values to hydromt instead of hourly. Could you kindly clear out the confusion.

**Comment by hboisgon** [comment, marked as answer]
To HydroMT what matters if that you give the required variables in the right units see [worflows.forcing.pet docs](https://deltares.github.io/hydromt/latest/_generated/hydromt.workflows.forcing.pet.html#hydromt.workflows.forcing.pet).

So (average) temperature in degree Celsius, pressure in hPa and radiations in W/m2 for PET De Bruin or Makkink.

If you give hourly values and wan to make daily forcing then hydromt will average temperature over the day and sum up radiations.
